README.md

```diff
@@ -45,11 +51,12 @@ Posts are backwards compatible, and shortcodes will still work. We are continuou
 
 ## The stages of Gutenberg
 
-Gutenberg has three planned stages. The first, aimed for inclusion in WordPress 5.0, focuses on the post editing experience and the implementation of blocks. This initial phase focuses on a content-first approach. The use of blocks, as detailed above, allows you to focus on how your content will look without the distraction of other configuration options. This ultimately will help all users present their content in a way that is engaging, direct, and visual.
+Gutenberg has three planned stages.
+1) **The first, aimed for inclusion in WordPress 5.0, focuses on the post editing experience** and the implementation of blocks. This initial phase focuses on a content-first approach. The use of blocks, as detailed above, allows you to focus on how your content will look without the distraction of other configuration options. This ultimately will help all users present their content in a way that is engaging, direct, and visual. These foundational elements will pave the way forward.
+2) Planned for 2019, **The second stage focuses on overhauling The Customizer** and page templates.
+3) Ultimately, **full site customization** will be possible.
```

Comment:
I don't know if we want to talk about the third phase and onwards yet, so it might be worth not mentioning it.

Reply:
Nothing to suggest here.